README.md

```diff
@@ -933,7 +933,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Fitbit](https://dev.fitbit.com/) | Fitbit Information | `OAuth` | Yes | Unknown |
 | [Football (Soccer) Videos](https://www.scorebat.com/video-api/) | Embed codes for goals and highlights from Premier League, Bundesliga, Serie A and many more | No | Yes | Yes |
 | [Football Prediction](https://boggio-analytics.com/fp-api/) | Predictions for upcoming football matches, odds, results and stats | `X-Mashape-Key` | Yes | Unknown |
-| [Football-Data.org](http://api.football-data.org/index) | Football Data | No | No | Unknown |
+| [Football-Data.org](http://api.football-data.org/index) | Football Data with match info, player, team, and competitions | `X-Mashape-Key` | Yes | Unknown |
```

Comment:
/

Reply:
No need to add the TLD ref. [contribution guide](https://github.com/public-apis/public-apis/blob/master/CONTRIBUTING.md)